doc/devel/coding_guide.rst

```diff
@@ -47,6 +47,18 @@ PR Review guidelines
 * If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
+* Tag the PR with appropriate labels.
+
+* Set the milestone to
+
+  * next point-release for new features.
+
+  * next bugfix release for bugfixes.
+
+  Primary PRs should usually still target the master branch. PRs milestoned
```

Comment:
What about "PRs should target the master branch unless [X,Y,Z] reason." I just worry that usually still might create uncertainty in the correct course of action.